### PR TITLE
Some documentation about editing

### PR DESCRIPTION
**Pull request by RenataMuellerC2C**

### CODE REVIEW

File: doc/administrator/editing.rst [hunk 1]
```diff
@@ -76,6 +78,45 @@ corresponding to this field is *Related Postgres table* in the admin interface.
...
+Configure layer editing
+-----------------------
+An overview of the metadata for the configuration of the editing can be found here:
+:ref:`administrator_administrate_metadata`.
+
+Registering edit time and user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* To register the time of the change of an edited item, a column of type
+  ``timestamp without timezone`` is needed in the table of the layer. In
+  the configuration of the layer ``lastUpdateDateColumn`` has to be added
```
Comment (jwkaltz) [inline doc/administrator/editing.rst:90]: This is also described in https://camptocamp.github.io/c2cgeoportal/2.6/administrator/administrate.html?highlight=lastupdatedatecolumn

Should it be removed from there, and a link provided there to this new section?

File: doc/administrator/editing.rst [hunk 2]
```diff
@@ -8,6 +8,8 @@ This section explains how to configure the edit function in c2cgeoportal.
 Just like most administrative tasks, setting up editing in a c2cgeoportal application involves intervening
 in the database, through the c2cgeoportal administration interface.
 
+.. _requirements:
```
Comment (sbrunner) [inline doc/administrator/editing.rst:11]:
```suggestion
.. _administrator_editing_requirements:
```

File: doc/administrator/editing.rst [hunk 3]
```diff
@@ -76,6 +78,45 @@ corresponding to this field is *Related Postgres table* in the admin interface.
...
 
+Configure layer editing
+-----------------------
+An overview of the metadata for the configuration of the editing can be found here:
+:ref:`administrator_administrate_metadata`.
+
+Registering edit time and user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* To register the time of the change of an edited item, a column of type
+  ``timestamp without timezone`` is needed in the table of the layer. In
```
Comment (sbrunner) [inline doc/administrator/editing.rst:89]: Are you sure about this? shouldn't be `timestamp with timezone`?

Comment (arnaud-morvan) [reply doc/administrator/editing.rst:89]: As I see, `timestamp with timezone` is not supported as the setted value is not timezone aware : https://github.com/camptocamp/c2cgeoportal/blob/c87b857bc2f08051ca514e3a37b2c32099e71ed5/geoportal/c2cgeoportal_geoportal/views/layers.py#L384

Comment (arnaud-morvan) [reply doc/administrator/editing.rst:89]: To be able to use a `timestamp with timezone` we would need `datetime.now(timezone.utc)`

**Review by sbrunner — APPROVED** [review]

**Pull request merged**